setup.cfg

```diff
@@ -43,3 +43,14 @@ ignore =
     F403
     E302
     F541
+
+[yapf]
```

Comment:
@glenn-jocher here is the configuration
feel free to edit according to your preference and let the bot apply changes :rabbit: